pkg/provider/provider.go

```diff
@@ -143,15 +147,35 @@ func (p *ExoscaleProvider) Configure(ctx context.Context, req provider.Configure
 
 	_ = clv2
 
+	// Exoscale v3 client
+	creds := credentials.NewStaticCredentials(
+		key,
+		secret,
+	)
+
+	opts := []exov3.ClientOpt{}
+	if ep := os.Getenv("EXOSCALE_API_ENDPOINT"); ep != "" {
```

Comment:
you're reusing `EXOSCALE_API_ENDPOINT` here, perhaps you can put it in a constant.